doc/storage/api/api.rst

```diff
@@ -9,7 +9,7 @@ Status codes
 
 The |API| uses the status code definitions that are shared with the other PSA Certified APIs.
 
-The following elements are defined in :file:`psa/error.h` from :cite-title:`PSA-STAT` (previously defined in :cite:`PSA-FFM`):
+The following elements are defined in :file:`psa/error.h` from :cite-title:`PSA-STAT` (previously defined in :cite:`PSA-FF-M`):
```

Comment:
This is another artifact of the initial file content not being fully synchronised with upstream.

Reply:
This has been broken again. Related to the previous commit about build errors which also undoes changes made in main in January,